internal/cluster/cluster_test.go

```diff
@@ -2905,82 +2905,124 @@ var _ = Describe("Transform day1 cluster to a day2 cluster", func() {
 	})
 
 	tests := []struct {
-		name          string
-		clusterStatus string
-		clusterKind   string
-		errorExpected bool
+		name               string
+		clusterStatus      string
+		clusterKind        string
+		errorExpected      bool
+		userManagedNetwork bool
+		apiVip             string
```

Comment:
```suggestion
		expectedAPIVipDNSName             string
```

Reply:
this should be ip as i need  away to verify case where apiVip is not set

Reply:
The test is literally just checking whether the APIVipDNSName field on the cluster is equal to this parameter, so the name of this parameter should be modified to emphasize that it's an expectation (i.e. "test output") and not a test "input" like the rest of the parameters in this struct

Reply:
Ah sorry I misread the test

Reply:
See https://github.com/openshift/assisted-service/pull/4191/files#r931196094 instead, it's the reason I got confused